docs/source/guide/configuration.rst

```diff
@@ -410,8 +410,7 @@ in the ``~/.aws/config`` file.
 ``tcp_keepalive``
     Toggles the TCP Keep-Alive socket option used when creating connections.
     By default this value is ``false``; TCP Keepalive will not be used
-    when creating connections. To enable TCP Keepalive with the system default configurations,
-    set this value to ``true``.
+    when creating connections. To enable TCP Keepalive with the system default configurations, set this value to ``true``.
```

Comment:
Was there a reason we made the line longer here? Ideally we'd keep length consistent and close to 80 characters or less.